viaf.py

```diff
@@ -717,15 +716,15 @@ def process_contributor(self, contributor):
             # No good match was identified.
             return None
 
-        (selected_candidate, match_confidences, contributor_titles) = best_contributor_candidate
+        (selected_candidate, match_confidences, contributor_titles) = contributor_candidate
         # Is there already another contributor with this VIAF?
         if selected_candidate.viaf is not None:
```

Comment:
Update #process_contributor to import VIAF information on the relevant contributors, according to its use in IdentifierResolutionCoverageProvider#resolve.viaf. (Update RedoOCLCForThreeMScript#do_run.)